content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -210,6 +210,10 @@ The following configuration variables are available **only** when using automati
 Enables or disables all automatic instrumentation. Setting the environment variable to `false` completely disables the CLR profiler. For other configuration methods, the CLR profiler is still loaded, but traces will not be generated. Valid values are: `true` or `false`.<br>
 **Default**: `true`
 
+`DD_DBM_PROPAGATION_MODE`
+: Enables DBM to APM link when set to `'service'` or `'full'`. The `'service'` option enables the connection between DBM and APM services. The `'full'` option enables connection between database spans with database query events. Available for Postgres and MySql.<br>
```

Comment:
Oh, also the casing we use for mysql is `MySQL`.

Reply:
Fixed in 69bfe6ece90c7bb51c1f11e37cc6e4d846f98d47